CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## 1.8.1
+Enhancements: 
+
+- Provides access to structs for building a Traffic Policy configuration
+
+Breaking changes:
+
+- Renames pre-release option `WithPolicyConfig` to `WithPolicyString`
+- Changes signature of `WithPolicy` option to accept the newly exposed structs for building a Traffic Policy configuration
```

Comment:
There shouldn't be breaking changes in a non-major version release, much less a patch version. We can get away with it if we have deprecated aliases for new names. The signature change is trickier, but we can make the new `Policy` struct implement `PolicyOption`.

We'll obviously also need to leave the `WithPolicy*` functions in place, but can add deprecations to them.

Reply:
What are you suggesting? a 4.0.0 major version release? just to be clear

Reply:
Agh yes, sorry meant to circle back on the versioning here. I did not think this was "right" but had mixed feelings since this was not a publicly available feature yet. I think I am in favor of making the leap to v2 here.

Reply:
I'd really prefer _not_ to bump to v2. It's something that has to be manually handled by users, changes import paths, etc. 

And we should think _very carefully_ about what breaking changes we want to introduce then, and try to do them all in one go rather than make a major bump for each one.

Reply:
Is anyone using this, it's not publicly available yet.

Reply:
[Yes it is.](https://pkg.go.dev/golang.ngrok.com/ngrok@v1.8.0/config#WithPolicy)

Reply:
It's currently behind a feature flag and undocumented.

Reply:
OK, I guess it's almost OK to break these then. But this could still occur:

* Someone updated to 1.8.0 between when it was cut an now
* We announce Policies and turn the feature flag on
* They think "oh cool, lemme try that out," and use the version they've already got in their go.mod.
* Time passes, and they run go module updates. Suddenly their previously-working code fails to compile.

Reply:
based on the conversation in slack, it sounds like we are making a very rare (never to be seen again!) exception here... are we okay with the current versioning? Any changes requested to verbiage in the changelog here?

Reply:
Nope, fine as-is